apps/vc-api/test/vc-api/workflows/resident-card/resident-card.e2e-suite.ts

```diff
@@ -166,7 +174,7 @@ export const residentCardWorkflowSuite = () => {
     const vp = await walletClient.provePresentation({ presentation, options: presentationOptions });
 
     // Holder submits presentation
-    await walletClient.continueWorkflowExchange(presentationExchangeContinuationEndpoint, vp, 'empty');
+    await walletClient.continueWorkflowExchange(presentationExchangeContinuationEndpoint, vp, 'redirectUrl');
```

Comment:
Should be kept as `empty`. Should pass after the `presentationDefinition` change from `didAuth` above.
```suggestion
    await walletClient.continueWorkflowExchange(presentationExchangeContinuationEndpoint, vp, 'empty');
```

Reply:
After changing to `presentationDefinition`:
The test case is expecting empty object : `{}`
But it is getting:
```
{\"verifiablePresentationRequest\":{\"challenge\":\"b555265b-a692-41e5-9756-ffe75e50a47f\",\"query\":[{\"type\":\"DIDAuth\",\"credentialQuery\":[{\"presentationDefinition\":{\"id\":\"286bc1e0-f1bd-488a-a873-8d71be3c690e\",\"input_descriptors\":[{\"id\":\"permanent_resident_card\",\"name\":\"Permanent Resident Card\",\"purpose\":\"We can only allow permanent residents into the application\",\"constraints\":{\"fields\":[{\"path\":[\"$.type\"],\"filter\":{\"type\":\"array\",\"contains\":{\"type\":\"string\",\"const\":\"PermanentResidentCard\"}}}]}}]}}]}],\"interact\":{\"service\":[{\"type\":\"UnmediatedHttpPresentationService2021\",\"serviceEndpoint\":\"http://localhost:3000/v1/vc-api/workflows/b229a18f-db45-4b33-8d36-25d442467bab/exchanges/57009a40-4e83-42ae-b08e-ee79f4d7ecf2\"}]},\"domain\":\"http://localhost:3000/v1/vc-api\"}}
```

Basically the `vpRequest`

Reply:
Ah yes, you're right. I think we fix the exchange creation and use `vpRequest` for now then. I've done this here: https://github.com/impactility/vc-api/pull/29